ui/pages/lock.tsx

```diff
@@ -133,9 +133,7 @@ export default function Lock() {
   useEffect(() => {
     if (hasLock && veNationLock) {
       setMinMaxLockTime({
-        min: dateToReadable(
-          oneWeekOut
-        ),
+        min: dateToReadable(bigNumberToDate(veNationLock[1])),
```

Comment:
What is the difference between `veNationLock[0]` and `veNationLock[1]`?

Reply:
It is weakly typed tuple, the first value `[0]` holds the amount the other `[1]` the date of the existing lock.

Reply:
@johnmark13 Ok, then maybe we can introduce a variable like `endDate = veNationLock[1]` or something like that, for increased code readability?

Reply:
I am happy too, but remember that this proposal was only a quick get out of jail before we refactor the whole tsx file and veNationLock[0|1] is all over the file.

Reply:
Made the change, local to the call, but can move to be a top level constant if you prefer?